README.md

```diff
@@ -36,21 +26,11 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
+👤 **Sameh Tantawy**
 
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
remove the unused parts in the readme file, professional documenting project is very important to employees.

Reply:
- [ ] [OPTIONAL] Great job giving reasons for the requested change. However, I suggest that you also phrase your comments politely and use suggestions.

### EXAMPLE
I think it will be better to remove the unused parts in the readme file because they do not make your work look neat